modules/API/API.hpp

```diff
@@ -23,6 +23,7 @@
 #include <memory>
 #include <mutex>
 #include <sstream>
+#include <list>
```

Comment:
to make clang-format happy, the includes need to be sorted alphabetically, so `<list>` needs to be above `<memory>`